diff --git a/node_modules/vue-drag-tree/src/DragNode.vue b/node_modules/vue-drag-tree/src/DragNode.vue
index 1c32c25..929c0fb 100644
--- a/node_modules/vue-drag-tree/src/DragNode.vue
+++ b/node_modules/vue-drag-tree/src/DragNode.vue
@@ -4,6 +4,7 @@
       <div :style="{ 'padding-left': (this.depth - 1) * 24 + 'px' }" :id='model.id' class='treeNodeText'>
         <slot :nodeName="model.name" :isClicked='isClicked'>
           <span :class="[isClicked ? 'nodeClicked' : '','vue-drag-node-icon']"></span>
+          
           <span class='text'>{{model.name}}</span>
         </slot>
       </div>
